source/clear-linux/get-started/bootable-usb/bootable-usb.rst

```diff
@@ -1,32 +1,203 @@
+   uncompressed image onto `/dev/sdd`:
+
+   .. code-block:: bash
+
+      dd if=./clear-[version number]-[image type] of=/dev/sdd bs=4M status=progress
+
+.. _bootable-usb-mac:
+
+Create a bootable USB drive on macOS*
+*************************************
+
+Make sure you have have completed all `Prerequisites`_.
+
+Before burning the image onto your USB drive,
+:ref:`verify and decompress your image <verify-mac>`.
+
+Burn the |CL| image onto a USB drive
+====================================
+
+.. caution::
+
+   |CAUTION-BACKUP-USB|
+
+#. Launch the Terminal app.
+#. Go to the directory with the decompressed image.
+#. Plug in a USB drive and get its identifier by entering the command
+   :command:`diskutil list`.  See Figure 1.
+
+   .. code-block:: console
+
+      diskutil list
+
+   .. figure:: figures/bootable-usb-mac-1.png
+      :scale: 100 %
+      :alt: Get USB drive identifier
+
+      Figure 1: macOS* - Get USB drive identifier
+
+#. Unmount the USB drive identified in the previous step.  The command-line
+   example below umounts `/dev/disk2`:
+
+   .. code-block:: console
+
+      diskutil umountDisk /dev/disk2
+
+#. Burn the image onto the drive using the :command:`dd` command.  The
+   command-line example below burns an uncompressed image onto `/dev/disk2`:
+
+   .. code-block:: console
+
+      sudo dd if=./clear-[version number]-[image type] of=/dev/rdisk2 bs=4m
+
+
+   Adding an ‘r’ in front of the disk identifier should help speed up the
+   imaging process.
+
+   You can press :kbd:`<CTL>-T` to check imaging progress.
+
+#. Eject the USB drive.
+
+   .. code-block:: console
+
+      diskutil eject /dev/disk2
+
+.. _bootable-usb-windows:
+
+Create a bootable USB drive on Windows\*
+****************************************
+
+Make sure you have have completed all `Prerequisites`_.
+
+Before burning the image onto your USB drive,
+:ref:`verify and decompress your image <verify-windows>`.
+
+Burn the |CL| image onto a USB drive
+====================================
+
+.. caution::
+
+   |CAUTION-BACKUP-USB|
+
+#. Download the `Rufus`_ utility to burn the image onto a USB drive.
+
+#. Plug in the USB drive and open Rufus.
+
+#. Click the :guilabel:`SELECT` button. See Figure 1.
+
+   .. figure:: figures/bootable-usb-windows-1.png
+      :scale: 80 %
+      :alt: Rufus utility - Click the SELECT button
+
+      Figure 1: Rufus utility - Click the SELECT button
+
+#. Find and select the previously extracted |CL| image file.
+   Then, click the  :guilabel:`Open` button. See Figure 2.
+
+   .. figure:: figures/bootable-usb-windows-2.png
+      :scale:  80 %
+      :alt: Rufus utility - Show and select |CL| image file
+
+      Figure 2: Rufus utility - Show and select |CL| image file
+
+#. Click the :guilabel:`START` button. See Figure 3.
+
+   .. figure:: figures/bootable-usb-windows-3.png
+      :scale: 80 %
+      :alt: Rufus utility - Click the START button
+
+      Figure 3: Rufus utility - Click START button
+
```

Comment:
All 3 of the figures in the Windows section seem gratuitous - am I missing something? 
Maybe just show the 3rd one to show all the filled-in fields.